src/simmi/assignment_16/FrequencyOfCharacter.java

```diff
@@ -0,0 +1,43 @@
+/*Assignment - 16 : 9th April'2021
+
+Find the frequency of character from a given String array.
+Note: Take String and character from the user.
+
+Input : technocredits
+char : e
+
+output : e -> 2*/
+
+
+package simmi.assignment_16;
+
+import java.util.Scanner;
+
+public class FrequencyOfCharacter {
+	String str;
+	char ch;
+	int count =0;
```

Comment:
You don't require to create this at instance level you can use this as local variable (I mean you should define this as local variable as this is useful to only single method).